### ISSUE TITLE
handlebars template sample renders as text

### ISSUE THREAD

**Issue by blittable** [seanmonstar/warp#348]
The response header is text/plain, so no great surprise.

Attempting to modify the header along these lines and didn't see a clear way to do it:
https://github.com/seanmonstar/warp/blob/8b8c9950260ef7312e29c4af72b0753619a5ad5c/src/reply.rs#L164

**Comment by jxs** [comment]
there are a couple ways to do it, but the easiest might be to just wrap the result of the render with `warp::reply::html`
```Rust
fn render<T>(template: WithTemplate<T>, hbs: Arc<Handlebars>) -> impl warp::Reply
where
    T: Serialize,
{
    let render = hbs.render(template.name, &template.value)
        .unwrap_or_else(|err| err.to_string());
    warp::reply::html(render)
}
```
want to submit a PR fixing the example?

**Comment by blittable** [comment]
Wonderful.  Works perfectly.   

I'll submit that PR.

Thanks.

**Comment by jxs** [comment]
closed with #349
